nano/node/common.hpp

```diff
@@ -22,40 +21,57 @@ bool parse_tcp_endpoint (std::string const &, nano::tcp_endpoint &);
 
 namespace
 {
+uint64_t random_64_m{ 0 };
+uint64_t const & random_64 ()
+{
+	if (random_64_m == 0)
+	{
+		random_64_m = (uint64_t) nano::random_pool::generate_word32 (0, 0xffffffffUL) << 32 | nano::random_pool::generate_word32 (0, 0xffffffffUL);
```

Comment:
Could we make a `nano::random_pool::generate_word64` ? That would lock the pool mutex only once and everything is in one place.

Reply:
Colin suggested 128 bit, then new function seems not required